web_group_expand/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Group Expand Buttons",
+    "category": "Web",
+    "version": "1.0",
+    "author": "OpenERP SA, "
+               "AvanzOSC, "
+               "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+               "Odoo Community Association (OCA)",
+    "depends": [
+        "web"
+    ],
+    "js": [
```

Comment:
As far as I know this has been deprecated since version 8 and does nothing. Replaced by extending `assets_backend`